src/Makevars.in

```diff
@@ -29,6 +29,8 @@ BBG_RPATH   = @rpath@
 
 ## set include and linker options
 ## Bbg API files are assummed to be in the standard search path
-PKG_CPPFLAGS = -I../inst/include/ -I.
```

Comment:
Why do you need the headers installed? They have no use at runtime? Some distributions don't even allow headers and bin's to be shipped in the same package.